src/overlays/actors/ovl_En_Jsjutan/z_en_jsjutan.c

```diff
@@ -168,7 +168,7 @@ void func_80A89A6C(EnJsjutan* this, GlobalContext* globalCtx) {
             actorBeanGuy = actorBeanGuy->next;
         }
 
-        // If somehow, neither of these two actors is not found, the game
+        //! @bug If somehow, neither of these two actors is not found, the game
         // will dereference a NULL pointer, but this is unlikely to happen.
```

Comment:
```suggestion
        //! @bug If somehow, either of these two actors is not found, the game
        // will dereference a NULL pointer, but this is unlikely to happen.
```
`neither` -> `either`

also don't think this can be considered a bug, as it absolutely cannot be achieved without modding and it's pretty obvious from the code that using this code outside its intended use won't work

Reply:
agreed, would probably just remove the comment all together personally